installation/setup.sh

```diff
@@ -44,8 +44,9 @@ source /etc/profile
 
 #setup go files
 mkdir $HOME/asic_q2
+cp env $HOME/asic_q2/.env
```

Comment:
asic_q2 is not representative to what the solution does. See comment above

Reply:
will do